packages/vscode-extension/lib/wrapper.js

```diff
@@ -209,6 +216,10 @@ export function PreviewAppWrapper({ children, ...rest }) {
       };
       LoadingView.hide = () => {
         devtoolsAgent._bridge.send("RNIDE_fastRefreshComplete");
+        if (isRunningPreview()) {
+          // refresh preview component
+          openPreview(getCurrentSceneName());
```

Comment:
I like all the other changes in this PR except for this one. It doesn't seem like the right choice to manually fix fast-refresh behavior. I wonder why does it in the main app but not in preview,  and would expect it to "just work" here as well? Maybe it's because preview is the root element and fast refresh doesn't support it? Maybe it'd be enough to just wrap it in some other root component?